bundles/org.openhab.binding.telegram/src/main/java/org/openhab/binding/telegram/bot/TelegramActions.java

```diff
@@ -1,507 +1,507 @@
+                if (!sendTelegramQuery(chatId, message, replyId, buttons)) {
+                    return false;
+                }
+            }
+        }
+        return true;
+    }
+
+    private boolean sendTelegramGeneral(@ActionInput(name = "chatId") @Nullable Long chatId, @Nullable String message,
+            @Nullable String replyId, @Nullable String... buttons) {
+        if (message == null) {
+            logger.warn("Message not defined; action skipped.");
+            return false;
+        }
+        if (chatId == null) {
+            logger.warn("chatId not defined; action skipped.");
+            return false;
+        }
+        TelegramHandler localHandler = handler;
+        if (localHandler != null) {
+            SendMessage sendMessage = new SendMessage(chatId, message);
+            if (localHandler.getParseMode() != null) {
+                sendMessage.parseMode(localHandler.getParseMode());
+            }
+            if (replyId != null) {
+                if (!replyId.contains(" ")) {
+                    if (buttons.length > 0) {
+                        InlineKeyboardButton[][] keyboard2D = new InlineKeyboardButton[1][];
+                        InlineKeyboardButton[] keyboard = new InlineKeyboardButton[buttons.length];
+                        keyboard2D[0] = keyboard;
+                        for (int i = 0; i < buttons.length; i++) {
+                            keyboard[i] = new InlineKeyboardButton(buttons[i]).callbackData(replyId + " " + buttons[i]);
+                        }
+                        InlineKeyboardMarkup keyBoardMarkup = new InlineKeyboardMarkup(keyboard2D);
+                        sendMessage.replyMarkup(keyBoardMarkup);
+                    } else {
+                        logger.warn(
+                                "The replyId {} for message {} is given, but no buttons are defined. ReplyMarkup will be ignored.",
+                                replyId, message);
+                    }
+                } else {
+                    logger.warn("replyId {} must not contain spaces. ReplyMarkup will be ignored.", replyId);
+                }
+            }
+            SendResponse retMessage = localHandler.execute(sendMessage);
+            if (!evaluateResponse(retMessage)) {
+                return false;
+            }
+            if (replyId != null && retMessage != null) {
+                logger.debug("Adding chatId {}, replyId {} and messageId {}", chatId, replyId,
+                        retMessage.message().messageId());
+                localHandler.addMessageId(chatId, replyId, retMessage.message().messageId());
+            }
+            return true;
+        }
+        return false;
+    }
+
+    @RuleAction(label = "Telegram message", description = "Sends a Telegram via Telegram API")
+    public boolean sendTelegram(@ActionInput(name = "chatId") @Nullable Long chatId,
+            @ActionInput(name = "message") @Nullable String format,
```

Comment:
Why did you change the parameter name?

Reply:
Can you comment here?

Reply:
@J-N-K which parameter name was changed?

Reply:
`message` to `format`. IMO this is not a good idea because

1) if no arguments are given, it's the message
2) the ActionInput is still named message, this is confusing

And please do not change the ActionInput name, this would be a breaking change and could not be merged to 2.5.x